js/freewheel.js

```diff
@@ -519,6 +519,7 @@ OO.Ads.manager(function(_, $) {
             _registerDisplayForLinearAd();
             fwContext.setParameter(tv.freewheel.SDK.PARAMETER_RENDERER_VIDEO_CLICK_DETECTION, false, tv.freewheel.SDK.PARAMETER_LEVEL_GLOBAL);
             slotStartedCallbacks[ad.ad.getCustomId()] = _.bind(function(ad) {
+              amc.focusAdVideo();
```

Comment:
Do we need to remove the previous call to `amc.focusAdVideo()` or was it not there before?

Reply:
It was not there before. It was done by the AMC after receiving notifyLinearAdStarted. There will be two calls to focusAdVideo, but focusing an already focused video element is fine. IMA already does this.

Reply:
Sounds good